fs/rails-base-graphql-api
# Fix AWS credentials check logic

### Summary

When we do not have AWS credentials the s3_options hash has empty string values. And `.all?` method works incorrectly and returns `true` value.

### How it works

N/A

### Test plan

- Remove AWS credentials from env
- Run project
- It should successfully started

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### Deploy notes

N/A
